book/src/simple-payment-and-state-verification.md

```diff
@@ -0,0 +1,153 @@
+# Simple Payment and State Verification
+
+Thin clients using Solana require a receipt for transactions that have been
+accepted by the cluster without running a full validator node.  New fullnode
+validators that desire to boot from a known checkpoint in the network require
+some proof that the checkpoint is valid.
+
+Solana receipts provide a proof that a transaction or state have been confirmed
+by the desired set of validators to a desired amount of finality.
+
+## Trusted Validators
+
+Thin clients send an RPC request to a validator to check for a
+signature confirmation.  The validator responds with a confirmation count for
+the signature, and clients are able to query the validator for the error code for
```

Comment:
only if the signature is relatively recent, though

is that a concern for this design?

Reply:
This just covers the trusted current implementation